test/core_tests.js

```diff
@@ -1,138 +1,156 @@
+    });
+
+    QUnit.test( "Attributes", function( assert ) {
+      assert.expect( 9 );
+
+      var actual, expected;
+      var canvas = document.querySelector( "div#impress > div" );
+
+      var canvasIsNotAStep = !canvas.classList.contains("step") && canvas.id === "";
+      assert.ok( canvasIsNotAStep, "Canvas do not have step element data" );
+
+      actual = canvas.style.transform;
+      expected = "rotateZ(0deg) rotateY(0deg) rotateX(0deg) translate3d(1000px, 0px, 0px)";
+      assert.strictEqual( actual, expected, "canvas.style.transform initialized correctly" );
+
+      actual = canvas.style.transformOrigin;
+      expected = "left top 0px";
+      assert.strictEqual( actual, expected, "canvas.style.transformOrigin initialized correctly" );
+
+      actual = canvas.style.transformStyle;
+      expected = "preserve-3d";
+      assert.strictEqual( actual, expected, "canvas.style.transformStyle initialized correctly" );
+
+      actual = canvas.style.transitionDelay;
+      expected = "0ms";
+      assert.strictEqual( actual, expected, "canvas.style.transitionDelay initialized correctly" );
+
+      actual = canvas.style.transitionDuration;
+      expected = "0ms";
+      assert.strictEqual( actual, expected, "canvas.style.transitionDuration initialized correctly" );
+
+      actual = canvas.style.transitionProperty;
+      expected = "all";
+      assert.strictEqual( actual, expected, "canvas.style.transitionProperty initialized correctly" );
+
+      actual = canvas.style.transitionTimingFunction;
+      expected = "ease-in-out";
+      assert.strictEqual( actual, expected, "canvas.style.transitionTimingFunction initialized correctly" );
+
+      actual = document.documentElement.style.height;
+      expected = "100%";
+      assert.strictEqual( actual, expected, "documentElement.style.height is 100%" );
+    });
+
+    QUnit.test( "Steps", function( assert ) {
+      assert.expect( 2 );
+
+      var actual, expected;
+      var step1 = document.querySelector( "div#step-1" );
+
+      actual = step1.style.position;
+      expected = "absolute";
+      assert.strictEqual( actual, expected, "Step position is 'absolute'" );
+
+      assert.ok( step1.classList.contains( "active" ), "Step 1 has active css class." );
+    });
+
+  QUnit.module( "Core API" );
+
+    QUnit.test( ".next()", function( assert ) {
+      assert.expect( 2 );
+
+      var root = document.querySelector( "div#impress" );
+      var step1 = document.querySelector( "div#step-1" );
+      var step2 = document.querySelector( "div#step-2" );
+      var done = assert.async();
+
+      impress().next();
+
+      assert.ok( step2.classList.contains( "active" ), step2.id + " add active css class." );
+      assert.notOk( step1.classList.contains( "active" ), step1.id + " remove active css class." );
+
+      // Reset to original state
+      var removeStepEnterEvent = registerEventListener( root, "impress:stepenter",  function() {
```

Comment:
This cleanup should be abstracted in a sandbox, because we are repeating the code in the next test and the cleanup s supposed to happen all the time to reset the test to the initial state. My idea is later to create some sort of "Proxy" for `QUnit.test()` so that it cleans up everything without having to manually do so in each test.